src/makeInfinityAdder.js

```diff
@@ -4,7 +4,26 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  let result = 0;
```

Comment:
The `result` variable is unnecessary. You can directly return `sum` when no arguments are provided.